src/brc/api/viewsets.py

```diff
@@ -4,6 +4,7 @@
 from zds_schema.utils import lookup_kwargs_to_filters
 from zds_schema.viewsets import NestedViewSetMixin
 
+from brc.api.scopes import SCOPE_BESLUITEN_ALLES_VERWIJDEREN
```

Comment:
I prefer relative imports within the same package, makes it easier to refactor/move code around.
```suggestion
from .scopes import SCOPE_BESLUITEN_ALLES_VERWIJDEREN
```